### PR TITLE
doc: update pre-launched VMs launched with offline ACPI

### PR DESCRIPTION
**Pull request by szhen11**
We use offline ACPI generator tool to create offline ACPI tables for
pre-launched VMs. Update getting start guide for hybrid and logical
partition mode to launch pre-launched VMs with offline ACPI binaries.

Signed-off-by: Shuang Zheng <shuang.zheng@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 1]
```diff
@@ -55,6 +56,8 @@ Perform the following to update Ubuntu GRUB so it can boot the hypervisor and lo
       file. The kernel command line arguments used to boot the Service VM are
       located in the header file ``misc/vm_configs/scenarios/hybrid/vm_configurations.h``
       and are configured by the `SOS_VM_BOOTARGS` macro.
+      The module ``/boot/ACPI_VM0.bin`` is the binary of ACPI tables for pre-launched VM0(Zephyr).
```
Comment (gvancuts) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst:59]:
```suggestion
      The module ``/boot/ACPI_VM0.bin`` is the binary of ACPI tables for pre-launched VM0 (Zephyr).
```

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 2]
```diff
@@ -55,6 +56,8 @@ Perform the following to update Ubuntu GRUB so it can boot the hypervisor and lo
...
+      The param ``ACPI_VM0`` is VM0's ACPI tag and should not be modified.
```
Comment (gvancuts) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst:60]:
```suggestion
      The ``ACPI_VM0`` parameter is VM0's ACPI tag and should not be modified.
```

**Review by gvancuts — APPROVED** [review]
The technical content looks good to me. I'll let @dbkinder do his pass for the language piece of the review.

Thanks!

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Review by dbkinder — APPROVED** [review]

**Pull request merged**